Makefile.in

```diff
@@ -51,13 +51,17 @@ else
 endif
 
 # If --with-arch is not specified, it defaults to whatever the compiler's
-# default is. The -with-abi is not necessary for this project. Unconditionally
-# compile it with a no-float ABI. i.e., ilp32 for 32-bit and lp64 for 64-bit.
+# default is.
 
 ifneq (@WITH_ARCH@,)
   march := -march=@WITH_ARCH@
-  is_32bit := $(findstring 32,$(march))
```

Comment:
Removing the `is_32bit` variable (perhaps) alters the logic a few lines below:

```
prefix       := @prefix@
enable_stow  := @enable_stow@

ifeq ($(enable_stow),yes)
  stow_pkg_dir := $(prefix)/pkgs
  INSTALLDIR ?= $(DESTDIR)/$(stow_pkg_dir)/$(project_name)-$(project_ver)
else
  ifeq ($(is_32bit),)
    install_subdir := @host_alias@
  else
    install_subdir := $(shell echo @host_alias@ | sed -e 's/64/32/g')
  endif
  INSTALLDIR ?= $(DESTDIR)/$(prefix)/$(install_subdir)
endif
```

In particular, the nested `else` branch is now never taken.

Reply:
OK, this might be problematic. The README says,

```
By default, 64-bit (RV64) versions of `pk` and `bbl` are built.  To
built 32-bit (RV32) versions, supply a `--with-arch=rv32i` flag to the
configure command.

The `install` step installs 64-bit build products into a directory
matching your host (e.g. `$RISCV/riscv64-unknown-elf`). 32-bit versions 
are installed into a directory matching a 32-bit version of your host (e.g.
`$RISCV/riscv32-unknown-elf`).
```

The question is what happens (and what should happen) when the user attempts to cross-compile PK for a RV32 host using the RV64 toolchain by just setting `--with-abi` and not providing `--host`. My original thinking was that setting `--host` was mandatory for cross-compilation... this is obviously not true.

Reply:
IIRC, `--host` is effectively required for compiling this package?

Reply:
The README is ambiguous. All of the example steps for building include `--host`, but `configure --help` suggests that it will be guessed (presumably, ultimately) from the `RISCV` environment variable.

```
System types:
  --build=BUILD     configure for building on BUILD [guessed]
  --host=HOST       cross-compile to build programs to run on HOST [BUILD]
```

Reply:
I think it's the usual configure thing where `host` will be guessed from the host machine.  In other words, the guess will be incorrect for sure, unless compiling on a native RISC-V system.